### DISCUSSION TITLE
index

### DISCUSSION THREAD

**Discussion by giscus** [plantree/press-comment#46] [General]
# index

小城市普通码农的生活

https://plantree.me/

**Comment by VictorGithub200** [comment]
想请教下这个博客网站的数据存储形式，以及是markdown转的html吗，文本编辑器是用的开源的吗？

**Reply by plantree** [reply]
网站由一个 Github repo 驱动，所有文章都保存为 Markdown 格式的文件，静态站点生成工具采用 Astro。

文本编辑器，用 VS Code，足够了。

**Reply by VictorGithub200** [reply]
很有价值的信息，感谢，最近也在写自己的博客，一开始用react加后端springboot服务，看了这篇文章后，准备考虑vercel加next.js搞了

**Reply by VictorGithub200** [reply]
https://blogofvictor.link，前段时间刚搭起来，班门弄斧了，后续会多向大佬学习，我也是工作了几年的程序猿

**Reply by plantree** [reply]
对于博客这种个人网站，轻交互，重的前端框架不是必要的，SSG 是我个人认为比较适合的技术选型。

仅供参考~
